spine/tests/SpineTest.cpp

```diff
@@ -94,8 +94,7 @@ class SpineTest : public ::testing::Test {
 
     params_.cpu = -1;         // no realtime scheduling
     params_.frequency = 400;  // Hz
-    params_.shm_name = std::string("/") + test_info->test_case_name() + "_" +
-                       test_info->name() + "_" + utils::random_string();
+    params_.shm_name = std::string("/") + utils::random_string();
```

Comment:
Original name exceeds PSHMNAMLEN in macOS, c.f. [https://opensource.apple.com/source/xnu/xnu-1699.22.81/bsd/kern/posix_shm.c.auto.html](here)

Reply:
Nice catch! :fishing_pole_and_fish: